tests/x11/x3270_ssl.pm

```diff
@@ -18,14 +18,28 @@ use strict;
 use warnings;
 use testapi;
 use utils;
+use power_action_utils 'power_action';
 
 sub run {
+
+    my ($self) = @_;
+
+    # Reboot the system when x11 launch fail to Workaround the Switch Console error
+    # (s390x limitation) s390x does not support snapshot to rollback the lastgood snapshot
+    # This workaround can avoid the blocked test fail from the last case
+    if (check_var('ARCH', 's390x')) {
+        if (check_var('FIPS_ENABLED', 1) && !get_var("FIPS_ENV_MODE")) {
+            power_action('reboot', textmode => 1);
+            $self->wait_boot(bootloader_time => 200);
+        }
+    }
+
+    select_console 'root-console';
+
```

Comment:
This issue is introduced by the fail of the previous test case.
The clean up can be done in the "post_fail_hook" either, maybe need no reboot, not sure as not try.
This method is OK too.

Reply:
Thanks for your comments, it would be good to do the enhancement in the future. :)